cmd/commands/cmd_import_mission_control.go

```diff
@@ -98,3 +101,161 @@ func importMissionControl(ctx *cli.Context) error {
 	_, err = client.XImportMissionControl(rpcCtx, req)
 	return err
 }
+
+var loadMissionControlCommand = cli.Command{
+	Name:     "loadmc",
+	Category: "Payments",
```

Comment:
This should go into `Mission Control`. And while we're at it, we should probably move `importmc` as well to that category.

Reply:
Nice catch, changed it :+1: